proto/substrait/plan.proto

```diff
@@ -43,3 +47,20 @@ message Plan {
   // one or more message types defined here are unknown.
   repeated string expected_type_urls = 5;
 }
+
+message Version {
+  // Substrait version number.
+  uint32 major = 1;
+  uint32 minor = 2;
+  uint32 patch = 3;
+
+  // If a particular version of Substrait is used that does not correspond to
+  // a version number exactly (for example when using an unofficial fork or
+  // using a version that is not yet released or is between versions), set this
+  // to the full git hash of the utilized commit of
+  // https://github.com/substrait-io/substrait (or fork thereof), represented
+  // using a lowercase hex ASCII string 40 characters in length. The version
+  // number above should be set to the most recent version tag in the history
+  // of that commit.
+  string git_hash = 4;
```

Comment:
Should the `major`/`minor`/`path` be left empty in this case (0.0.0)?  Or should they be set to the most recent version before the fork?  I would think empty but might be good to clarify.

Reply:
Doesn't the last sentence of the comment cover it? I guess I could make explicit that the tag I'm referring to must be an "official" substrait-io version tag.

Reply:
Ah, yeah, that is clear.  I think I just glossed over it reading, sorry